python/pyarrow/tests/parquet/test_dataset.py

```diff
@@ -1137,7 +1137,9 @@ def _make_dataset_for_pickling(tempdir, N=100):
 
 
 @pytest.mark.pandas
-def test_pickle_dataset(tempdir, pickle_module):
+def test_pickle_dataset(tempdir, pickle_module, request):
+    pickle_module = request.getfixturevalue(pickle_module)
```

Comment:
Does it work to move this `getfixturevalue` inside the `pickle_module` fixture?

Reply:
Works, much nicer like that =) will change!